holidays/countries/falkland_islands.py

```diff
@@ -0,0 +1,128 @@
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays import GOVERNMENT, PUBLIC
+from holidays.calendars.gregorian import SEP
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_MON,
+    SUN_TO_NEXT_WED,
+    SAT_TO_NEXT_MON,
+    SUN_TO_NEXT_TUE,
+    SAT_TO_NEXT_TUE,
+)
+
+
+class FalklandIslands(
+    ObservedHolidayBase, ChristianHolidays, InternationalHolidays, StaticHolidays
+):
+    """Falkland Islands holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_the_Falkland_Islands>
+        * <https://web.archive.org/web/20241201174202/https://www.timeanddate.com/holidays/falkland-islands/2025>
+        * [Falkland Day](https://web.archive.org/web/20250325025508/https://www.daysoftheyear.com/days/falklands-day/)
+        * [Peat Cutting Day](https://en.wikipedia.org/wiki/Peat_Cutting_Monday#:~:text=Peat%20Cutting%20Day%20is%20a,Monday%20in%20October%20every%20year.)
+        * [King's Birthday](https://web.archive.org/web/20230329063159/https://en.mercopress.com/2022/11/03/falklands-appoints-14-november-as-a-public-holiday-to-celebrate-birthday-of-king-charles-iii)
+        * [2023-2026](https://web.archive.org/web/20250402143330/https://www.falklands.gov.fk/policy/downloads?task=download.send&id=186:falkland-islands-public-holidays-2023-2026&catid=12)
+    """
+
+    country = "FK"
+    default_language = "en_GB"
+    supported_languages = ("en_GB", "en_US")
+    supported_categories = (GOVERNMENT, PUBLIC)
+    # %s observed.
+    observed_label = tr("%s (observed)")
+
+    start_year = 1983
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=FalklandIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        if self._year <= 2022:
+            # Queen's Birthday.
+            self._add_observed(self._add_holiday_apr_21(tr("Queen's Birthday")))
+
+        # Liberation Day.
+        self._add_observed(self._add_holiday_jun_14(tr("Liberation Day")))
+
+        if self._year <= 2001:
+            # Falkland Day.
+            self._add_observed(self._add_holiday_aug_14(tr("Falkland Day")))
+        else:
+            # Peat Cutting Day.
+            self._add_holiday_1st_mon_of_oct(tr("Peat Cutting Day"))
+
+        if self._year >= 2022:
+            # King's Birthday.
+            self._add_observed(self._add_holiday_nov_14(tr("King's Birthday")))
```

Comment:
Official name listed in the law + local gov website
```suggestion
            # HM The King's Birthday.
            self._add_observed(self._add_holiday_nov_14(tr("HM The King's Birthday")))
```